### DISCUSSION TITLE
Is it possible export the charts?

### DISCUSSION THREAD

**Discussion by vito-gmi** [ecomfe/vue-echarts#666] [General]
Hi, vue-echarts devs,

I am curious if it is possible somehow to export the charts in a file, e.g., a word document? I will be very happy if I can find a solution, because I am was trying some various ways, but nothing works. Thanks!

BR,
Vitomir

**Comment by Justineo** [comment]
See the convert to image feature [here](https://vue-echarts.dev/#map).

[Source code](https://github.com/ecomfe/vue-echarts/blob/main/src/demo/Demo.vue#L458-468)

**Reply by vito-gmi** [reply]
Thanks for the answer! So, you suggest to convert it that way and put it inside the word, right?

Currently, I have issue with the word document creation, hopefully, will have it working soon and will try to insert this image. If you know somewhere such a code, I will be happy to see it.
